finngeorge/inviteonlycatalog
# Create database models

## Addresses
- [Create database models](https://app.asana.com/0/1207201509287914/1207201509287923/f)

## Summary
- Updates `README.md`
- Removes cached `*.pyc` files from previous merge (oops)
- Updates `.gitignore`
- Install `Pillow` and add it to `requirements.txt` file
- Run migration to create initial tables

## Testing
_Please see Readme for clarification..._

- Create and spin up a virtual environment
- Install requirements modules: `pip install -r requirements.txt`
- Run `python manage.py migrate`
- Query any of the `inviteonlydb_...` tables in pgAdmin and observe expected columns present
